crates/sui-source-validation-service/src/main.rs

```diff
@@ -69,6 +82,11 @@ pub async fn main() -> anyhow::Result<()> {
     let server = tokio::spawn(async { serve(app_state_copy)?.await.map_err(anyhow::Error::from) });
     threads.push(server);
 
+    let metrics_listener = std::net::TcpListener::bind(METRICS_HOST_PORT)?;
+    let registry_service = start_prometheus_server(metrics_listener);
+    let prometheus_registry = registry_service.default_registry();
+    prometheus_registry.register(mysten_metrics::uptime_metric(VERSION, "sui-source-service"))?;
```

Comment:
From the expect test output, I don't think you want to use this exact metric, because it's about the uptime of the chain (the name you've given is being interpreted as a "chain identifier") which is usually the prefix of the chain's first checkpoint's digest.

Reply:
good catch, this isn't what we should use (I somehow though `uptime_metric` was more general but turns out it's not). I've gone ahead and implemented a request counter instead of this https://github.com/MystenLabs/sui/pull/14448/commits/44324abff711f66ac4c9e975b8df38a8fba9141b